mypyc/test-data/run-strings.test

```diff
@@ -178,3 +148,164 @@ def test_str_to_bool() -> None:
     for x in 'a', 'foo', 'bar', 'some string':
         assert is_true(x)
         assert not is_false(x)
+
+[case testFormattedStringLiterals]
```

Comment:
Nit: Maybe call this `testFStrings`, since that's the most commonly used name for the feature.